README.md

```diff
@@ -73,6 +73,65 @@ We also add following methods to make type-checking more easily:
 - [`find_n`, `first_n`, `last_n`](https://github.com/chanzuckerberg/sorbet-rails#find-first-and-last)
 - [`pluck_to_tstruct`](#pluck_to_tstruct-instead-of-pluck)
 
+#### Enums
+
+If you use [Rails `enum`](https://guides.rubyonrails.org/active_record_querying.html#enums), `sorbet-rails` will generate a corresponding `T::Enum`. It will also include getters, setters, and scope methods in the rbi file it generates.
+
+ie.
+
+```ruby
+  enum house: {
+    Gryffindor: 0,
+    Hufflepuff: 1,
+    Ravenclaw: 2,
+    Slytherin: 3,
+  }
+```
+
+Will generate this enum:
+
+```ruby
+class Wizard::House < T::Enum
+  enums do
+    Gryffindor = new('Gryffindor')
+    Hufflepuff = new('Hufflepuff')
+    Ravenclaw = new('Ravenclaw')
+    Slytherin = new('Slytherin')
+  end
+end
+```
+
+And these methods:
+
+```ruby
+  sig { returns(T.nilable(String)) }
+  def house; end
```

Comment:
we will also generate `typed_house` here.

Reply:
@ghiculescu do you think we should tell people that `typed_house` are added by default?